content/blog/glc-scim-api-integration-with-okta-scim-adapter-1.md

```diff
@@ -0,0 +1,5933 @@
+---
+title: GLC SCIM API Integration with Okta SCIM Adapter
```

Comment:
Namik, I guess GLC means HPE GreenLake Central. Correct?

Abbreviation should not be used for everything related to HPE GreenLake in title and throughout your blog post.
Also, Title should be sentence case.
So the title should be:

HPE GreenLake Central SCIM API integration with Okta SCIM adapter

Having said that, HPE GreenLake Central is now called "HPE GreenLake Flex Solutions". So you may want to mention HPE GreenLake Flex Solutions for the Title and throughout your post.

If you have access to a HPE GreenLake workspace, and go to Service Catalog, you will see the description below for HPE GreenLake Flex Solutions:

Overview
**Formerly known as HPE GreenLake Central**, view usage and cost of HPE GreenLake Flex Solutions and manage the Private Cloud Enterprise and High Performance Computing cloud services. For all HPE GreenLake Flex Solutions, this service provides reporting on billing, active capacity planning, and powerful consumption analytics that enable transparency to see costs by service type, department, or other parameters, including public cloud costs. Customers of Private Cloud Enterprise use it to provision and manage virtual, containerized, and bare metal resources, and customers of High Performance Computing use it to operate HPC workloads.